Projects/Home/Sources/HomeState.swift

```diff
@@ -0,0 +1,110 @@
+import Apollo
+import Flow
+import Presentation
+import hCore
+import hGraphQL
+
+public struct HomeState: StateProtocol {
+	var memberContractState: MemberContractState = .loading
```

Comment:
Should home know what a contract is? Maybe there could be a more generic name for this?